server/src/main/java/com/distkv/dst/server/DstServerConfig.java

```diff
@@ -0,0 +1,81 @@
+package com.distkv.dst.server;
+
+import com.distkv.drpc.config.ServerConfig;
+
+public class DstServerConfig {
+  private int listeningPort;
+  private int workerThreadNum;
+  private boolean isMaster;
+  private int shardNum;
+
+  public boolean isMaster() {
+    return isMaster;
+  }
+
+  public ServerConfig genRpcConfig() {
+    ServerConfig serverConfig = ServerConfig.builder()
+        .port(listeningPort)
+        .workerThreadNum(workerThreadNum)
+        .build();
+    return serverConfig;
+  }
+
+  public int getPort() {
+    return  listeningPort;
+  }
+
+  public int getShardNum() {
+    return shardNum;
+  }
+
+  DstServerConfig(int listeningPort,
+                  int workerThreadNum,
+                  boolean isMaster,
+                  int shardNum) {
+    this.listeningPort = listeningPort;
+    this.isMaster = isMaster;
+    this.workerThreadNum = workerThreadNum;
+    this.shardNum = shardNum;
+  }
+
+  public static DstServerConfig.DstServerConfigBuilder builder() {
+    return new DstServerConfig.DstServerConfigBuilder();
+  }
+
+  public static class DstServerConfigBuilder {
```

Comment:
We should add a static method to create this Config instance instead of build mode:
```java
Config config = ConfigFactory.systemProperties();
config.withFallback(ConfigFactory.load(DEFAULT_CONFIG_FILE);
```

So that we needn't to set the items one by one.